salt/utils/network.py

```diff
@@ -1896,8 +1896,16 @@ def refresh_dns():
         pass
 
 
+@jinja_filter('connection_check')
+def connection_check(addr, port=80, safe=False, ipv6=None):
+    '''
+    Provides a convenient alias for the dns_check filter.
+    '''
+    return dns_check(addr, port, safe, ipv6)
+
+
 @jinja_filter('dns_check')
-def dns_check(addr, port, safe=False, ipv6=None):
+def dns_check(addr, port=80, safe=False, ipv6=None):
```

Comment:
Should this default to `None` instead of 80?

Also, this change should be tested.

Reply:
Added a test case to the dictionary that does not include the port.  Updated the call to dns_check to not pass port if it's not in the dictionary.  Updated the default to be None.